### ISSUE TITLE
"EPISODES NOT PLAYING OR LOADING"

### ISSUE THREAD

**Issue by skgupta507** [viplove3541/animez-2.0#1]
Hey @viplove3541 the website is showing the anime but its not loading the episodes nor the video player can  fix this pls

**Comment by anime-kun32** [comment]
yep even now it still loading the episodes , it just saying hang in there
